package.json

```diff
@@ -51,7 +50,7 @@
     "postcss-safe-parser": "4.0.1",
     "react": "^16.8.6",
     "react-app-polyfill": "^1.0.1",
-    "react-dev-utils": "^11.0.4",
+    "react-dev-utils": "^9.0.0",
```

Comment:
Why roll this back to an older version?

Reply:
I was receiving an error and it was suggested on a thread to downgrade to 9. I am open to suggestions!

Emitted 'error' event on Socket instance at: at emitErrorNT (internal/streams/destroy.js:106:8) at processTicksAndRejections (internal/process/task_queues.js:80:21)

https://github.com/facebook/create-react-app/issues/9804

Reply:
There are several security issues with 9.0.0, but if you could set it to 9.0.7 instead i believe it should be ok.

Reply:
yes i am building on windows

Reply:
Ok, let see if 9.0.7 solves it :)

Reply:
yup 9.0.7 looks good! Updated!